app/scripts/modules/canary/canary/canaryScore.component.less

```diff
@@ -1,4 +1,4 @@
-@import (reference) '~core/presentation/less/imports/commonImports.less';
+@import (reference) '../../core/src/presentation/less/imports/commonImports.less';
```

Comment:
we'll probably want to figure out a better way to manage these cross-package imports.